src/app/forms-builder/components/forms-builder-drop-area/forms-builder-drop-area.component.html

```diff
@@ -0,0 +1,40 @@
+<ng-template [cdkPortalOutlet]="dropAreaPortal"></ng-template>
+
+<ng-template #dropAreaPortalContent>
+  <div
+    id="drop-list"
+    cdkDropList
+    #dropList
+    [cdkDropListData]="dropAreaState"
+    class="example-list"
+    (mouseenter)="enterToDropArea(dragAreaState.isDragging)"
+    (mouseleave)="leaveDropArea()"
+    [style]="
+      accordionState.formStyling
+        | stylePipe
+          : [
+              'width',
+              'height',
+              'padding',
+              'borderWidth',
+              'borderStyle',
+              'borderColor',
+              'borderRadius',
+              'backgroundColor'
+            ]
```

Comment:
use array in .ts file instead
cssProperties = [
              'width',
              'height',
              'padding',
              'borderWidth',
              'borderStyle',
              'borderColor',
              'borderRadius',
              'backgroundColor'
            ]
```suggestion
          : cssProperties
```